andela/merry-ah-frontend
# #162727418 User should reset password

#### What does this PR do?
- This PR allows users to reset their password

#### Description of Task to be completed?
- Create a screen to allow users to input their email address to get password reset links
- Create a screen for password reset
- Implement password reset logic

#### How should this be manually tested?
- Install dependencies with `npm i`
- Start the development server with `npm run start:dev`
- Navigate to `http://localhost:5050/forgot-password` in the browser to see email verification field
- Once a link has been sent, go to the mailbox and click on link to navigate to password reset screen.

#### Any background context you want to provide?
- The user must be signed up on the platform for the request to be completed

#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/162727418

#### Screenshots (if appropriate)
<img width="1440" alt="screen shot 2019-02-15 at 1 24 42 pm" src="https://user-images.githubusercontent.com/42177767/52856609-312b8f00-3125-11e9-9170-6b8006b31acf.png">
<img width="1440" alt="screen shot 2019-02-15 at 1 24 51 pm" src="https://user-images.githubusercontent.com/42177767/52856611-312b8f00-3125-11e9-9538-ab315473c631.png">

#### Questions:
N/A
